docs/runtime-agent/02-01-runtime-agent-workflow.md

```diff
@@ -1,20 +1,19 @@
 ---
-title: Runtime Agent architecture
-type: Architecture
+title: Architecture
 ---
 
-This document presents the workflow of the Runtime Agent. 
+This document presents the workflow of the Runtime Agent.
 
 ![Runtime Agent architecture](./assets/runtime-agent-architecture.svg)
 
-1. Runtime Agent fetches the certificate from the Connector to initialize connection with Compass.
+1. Runtime Agent fetches the certificate from the [Connector](https://github.com/kyma-incubator/compass/blob/master/docs/connector/02-01-connector-service.md) to initialize connection with Compass.
```

Comment:
I was thinking: do we want to make the `initialize connection with Compass` bit to link to either [`03-01-connection-with-compass.md`](https://github.com/kyma-project/kyma/blob/007037bd803ee50badf576d61a61360cdbb33271/docs/runtime-agent/03-01-connection-with-compass.md) or [`08-07-establish-secure-connection-with-compass.md`](https://github.com/kyma-project/kyma/blob/007037bd803ee50badf576d61a61360cdbb33271/docs/runtime-agent/08-07-establish-secure-connection-with-compass.md)? Just a thought.

Reply:
I'm afraid it will be too packed with links if we do so 😕